package/openvpn/Config-mbedtls.in

```diff
@@ -1,4 +1,4 @@
-if PACKAGE_openvpn-mbedtls
+if PACKAGE_openvpn-gargoyle-mbedtls
 
 config OPENVPN_mbedtls_ENABLE_LZO
```

Comment:
Conflict with standard openvpn package. IMO build option also should be changed to OPENVPN_GARGOYLE_mbedtls etc.

Reply:
mmm. Much more work. 
We don’t do this for mjpg-streamer... is still a big deal?

Reply:
mjpg-streamer come from feeds, not core packages.

If you change openvpn name, build script leaves original openvpn in package/network/services. It can make some interferences in build process (I think so).

Reply:
You are right. Ok I agree, will fix this evening.

Reply:
New changes look ok?

Reply:
Looks good.